src/test/scala/org/zalando/nakadi/client/KlientSpec.scala

```diff
@@ -74,44 +46,29 @@ class KlientSpec extends WordSpec with Matchers with BeforeAndAfterEach with Laz
     }
   }
 
-  private def performStandardRequestChecks(expectedRequestPath: String, expectedRequestMethod: HttpString) = {
+  private def performStandardRequestChecks(expectedRequestPath: String, expectedRequestMethod: HttpString): NakadiRequest = {
     val collectedRequestsMap = service.getCollectedRequests
     val requests = collectedRequestsMap.get(expectedRequestPath)
     requests should not be null
 
     val request = Iterators.getLast(requests.iterator)
-    request.getRequestPath should be(expectedRequestPath)
-    request.getRequestMethod should be(expectedRequestMethod)
+      //
+      request.getRequestPath shouldBe expectedRequestPath
```

Comment:
I slightly prefer `shouldBe X` over `should be (x)` since it does not require the paranthesis.